javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,48 @@
+//A future age calculator
+
+let yearOfBirth = 1982;
+let yearFuture = 2046;
+let age;
+age = yearFuture - yearOfBirth;
+console.log("\"You will be " + age + " years old in " + yearFuture + "\"");
+
+//A dog age calculator
+
+let dogYearOfBirth = 2000;
+let dogYearFuture = 2010;
+let dogYear = (dogYearFuture - dogYearOfBirth) * 4;
+let shouldShowResultInDogYears;
+if (shouldShowResultInDogYears = true) {
+    console.log("\"Your dog will be " + dogYear + " dog years old in " + dogYearFuture + "\"");
+} else {
+    console.log("afe\"Your dog will be " + (dogYearFuture - dogYearOfBirth) + " human years old in " + dogYearFuture + "\"")
+}
+
+//Housey Pricey
+
+let housePrice;
+let volumInMeters;
+let gardenSizeInM2;
+volumInMeters = 8 * 10 * 10;
+gardenSizeInM2 = 100;
+housePrice = volumInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
```

Comment:
Since we are declaring out variable and assigning them some values right after, it might be worth just doing the declaration and initialization in a single line. eg:

```
let gardenSizeInM2 = 100;
let volumeInMeters = 8 * 10  * 10
// etc ...
```

Same result really but we'll have a little less lines of code